Bio/PDB/PDBList.py

```diff
@@ -701,88 +702,166 @@ def get_seqres_file(self, savefile="pdb_seqres.txt"):
-                    overwrite = True
-                elif option in ("-pdb", "-xml", "-mmtf"):
-                    file_format = option[1:]
-                # Allow for download of assemblies alongside ASU
-                elif option == "-with-assemblies":
-                    with_assemblies = True
 
-    else:
-        pdb_path = os.getcwd()
-        pl = PDBList()
-        pl.flat_tree = True
-
-    if len(sys.argv) > 1:
-        if sys.argv[1] == "update":
-            # update PDB
-            print("updating local PDB at " + pdb_path)
-            pl.update_pdb(file_format=file_format, with_assemblies=with_assemblies)
+    desc = """
+    (c) Kristian Rother 2003, Wiktoria Karwicka & Jacek Smietanski 2016
+    Contributed to Biopython
 
-        elif sys.argv[1] == "all":
-            # get the entire PDB
-            pl.download_entire_pdb(file_format=file_format)
-            if with_assemblies:
-                # get all assembly structures
-                pl.download_all_assemblies(file_format=file_format)
+    Quick access to the structure lists on the PDB or its mirrors.
+    Requires a mode (update/all/obsol/assemb/single/multi) to be specified.
+    By default (no format specified) structures are downloaded as PDBx/mmCif files.
+    """
 
-        elif sys.argv[1] == "obsol":
-            # get all obsolete entries
-            pl.download_obsolete_entries(pdb_path, file_format=file_format)
+    # Instantiate argument parser/subparser
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter, description=desc
+    )
+
+    # Add positional argument for pdb_path
+    parser.add_argument(
+        "pdb_path",
+        nargs="?",
+        default=os.getcwd(),
+        help="local path to use, defaults to the current working directory",
+    )
+
+    # Add mutually exclusive arguments for mode
+    mode = parser.add_mutually_exclusive_group(required=True)
+    mode.add_argument(
+        "-update",
+        action="store_true",
+        help="write weekly PDB updates to local pdb tree",
+    )
+    mode.add_argument(
+        "-all", action="store_true", help="write all PDB entries to local pdb tree"
+    )
+    mode.add_argument(
+        "-obsol",
+        action="store_true",
+        help="write all obsolete PDB entries to local pdb tree",
+    )
+    mode.add_argument(
+        "-assemb",
+        action="store_true",
+        help="write all assemblies for each PDB entry to local pdb tree",
+    )
+    mode.add_argument(
+        "-single", help="retrieve single structure - accepts a single PDB code"
+    )
+    mode.add_argument(
+        "-multi",
+        help="retrieve a set of structures - accepts a comma separated list of PDB codes",
+    )
+
+    # Add mututally exclusive arguments for format
+    format = parser.add_mutually_exclusive_group()
+    format.add_argument(
+        "-pdb", action="store_true", help="downloads structures in PDB format"
+    )
```

Comment:
I think we can swap to:
```suggestion
    format.add_argument(
        "-pdb", action='store_const', const="pdb", dest="format", help="downloads structures in PDB format"
    )
```

to avoid the if/else L794

Reply:
This was an excellent idea, but I instead just included an optional `-format` argument with the choice pdb, xml, or mmtf with a default value of mmCif.